docs/orchestration/faq/faq.md

```diff
@@ -0,0 +1,228 @@
+
+On the Prefect side, the native Python logger gets serialized and then sent to workers. When it gets deserialized, it loses the configuration attached, so the logs are not directed to Prefect Cloud.
+
+There are several ways you could approach this. 
+
+1. You could set up some log aggregation service to send Dask worker logs for debugging.
+2. You can view the worker logs directly.
+
+### [REVIEW] Is there an equivalent to Airflow sensors in Prefect? How do I trigger event-driven workflows?
+
+Overall, there are a couple of ways how you could tackle it:
+
+1. Event based: once the event that you are looking for occurs, then trigger a specific flow to run - if you are on AWS, this event could trigger a Lambda function that triggers a Flow run. The event could be a new file arrived in S3 or a new row streamed to DynamoDB/Kinesis/AWS Aurora.
+2. API-based: since starting a FlowRun is just an API call, you can do that from any flow or even a task, there is a lot what you could do that, e.g. as you mentioned one table got updated in a task, and subsequent task triggers some other flow or makes an API call to take action
+3. State based: this would work in a similar way to Airflow sensors - you raise a RETRY signal if the condition you poll for is not met.
+
+```python
+import pendulum
+from prefect.engine.signals import RETRY
+
+@task
+def my_sensor(**kwargs):
+    # check some state of the world
+    if condition_not_met:
+        raise RETRY("Condition not met, retrying in 5 seconds.", start_time=pendulum.now().add(seconds=5))
+```
+
+### How do I set a dynamic default value for a Parameter like a callable?
+
+If you do something like:
+
+```python
+from datetime import date
+
+with Flow("example") as flow:
+	start_date = Parameter("start_date", default=(date.today().strftime("%Y-%m-%d"))
+	do_something(start_date)
+```
+
+The default Parameter value is evaluated during registration time, so the start_date in the example above will be fixed to the registration date. In order to add dynamicism, using a task is needed to defer the execution of date.today() to runtime. For example:
+
+```python
+from datetime import date
+
+@task
+def get_date(day):
+    if day == None:
+        return date.today().strftime("%Y-%m-%d")
+    else:
+        return day
+
+with Flow("example") as flow:
+	start_date = Parameter("start_date", default=None)
+    new_date = get_date(start_date)
+	do_something(new_date)
+```
+
+### How can I set a custom flow run name?
+
+The flow run name can not be set in advance, but it can be changed using the [RenameFlowRun](api/latest/tasks/prefect.html#renameflowrun) task after the flow run has been created. You can use this task inside the Flow block, or through a flow-level state handler. When calling the task from a state handler, make sure to call the task’s  `.run()` method.
```

Comment:
```suggestion
The flow run name cannot be set in advance, but it can be changed using the [RenameFlowRun](api/latest/tasks/prefect.html#renameflowrun) task after the flow run has been created. You can use this task inside the flow block, or through a flow-level state handler. When calling the task from a state handler, make sure to call the task’s  `.run()` method.
```

Reply:
The RenameFlowRun URL may need to be adjusted as well